quaint/.gitignore

```diff
@@ -3,3 +3,7 @@
 Cargo.lock
 .direnv/
 .vscode/
+
+docker-compose.override.yml
+db/
```

Comment:
I think that did not work as the PR includes that file, possible?

Reply:
ah, test.db is included, that's weird